site/no-nb/index.html

```diff
@@ -0,0 +1,669 @@
+                </div>
+                <div class="col-lg-7 col-sm-6">
+                    <ul class="nav nav-tabs">
+                        <li class="active"><a href="#maven" data-toggle="tab">Maven</a></li>
+                        <li><a href="#gradle" data-toggle="tab">Gradle</a></li>
+                        <li><a href="#ivy" data-toggle="tab">Ivy</a></li>
+                        <li><a href="#download" data-toggle="tab">Nedlasting</a></li>
+                    </ul>
+                    <div id="dependencies" class="tab-content">
+                        <div class="tab-pane fade in active" id="maven">
+                            <pre  class="prettyprint">
+&lt;dependency&gt;
+  &lt;groupId&gt;org.eclipse.collections&lt;/groupId&gt;
+  &lt;artifactId&gt;eclipse-collections-api&lt;/artifactId&gt;
+  &lt;version&gt;10.4.0&lt;/version&gt;
+&lt;/dependency&gt;
+
+&lt;dependency&gt;
+  &lt;groupId&gt;org.eclipse.collections&lt;/groupId&gt;
+  &lt;artifactId&gt;eclipse-collections&lt;/artifactId&gt;
+  &lt;version&gt;10.4.0&lt;/version>
+&lt;/dependency&gt;</pre>
+                        </div>
+                        <div class="tab-pane fade" id="gradle">
+                            <pre  class="prettyprint">
+implementation 'org.eclipse.collections:eclipse-collections-api:10.4.0'
+implementation 'org.eclipse.collections:eclipse-collections:10.4.0'</pre>
+                        </div>
+                        <div class="tab-pane fade" id="ivy">
+                            <pre  class="prettyprint">
+&lt;dependency org="org.eclipse.collections" name="eclipse-collections-api" rev="10.4.0" /&gt;
+&lt;dependency org="org.eclipse.collections" name="eclipse-collections" rev="10.4.0" /&gt;</pre>
+                        </div>
+                        <div class="tab-pane fade" id="download">
+                            <pre class="prettyprint">
+<a target="_blank" href="http://mvnrepository.com/artifact/org.eclipse.collections/eclipse-collections-api/10.4.0">eclipse-collections-api-10.4.0.jar</a>
+<a target="_blank" href="http://mvnrepository.com/artifact/org.eclipse.collections/eclipse-collections/10.4.0">eclipse-collections-10.4.0.jar</a></pre>
+                        </div>
+
+                    </div>
+                </div>
+            </div>
+
+        </div>
+        <!-- /.container -->
+
+    </div>
+    <!-- Get Started -->
+
+    <a class="anchor" name="concept"></a>
+    <div class="content-section-b">
+
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-push-7 col-lg-5 col-sm-push-6  col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">Rikelige, konsise og lesbare API-er</h2>
+                    <p class="lead">Maksimer kraften til lambda-uttrykk og metodereferanser i Java 8 med rikelige API-er 
```

Comment:
Somewhat unsure on this one - is "omfattende" more suitable than "rikelige"?

Reply:
Not really sure about this one... I kind of feel "omfattende" has a lightly more of a negative cling to it? WDYT, @madsop?

Reply:
Maybe _"omfangsrike"_?